src/ensembl/src/content/app/browser/drawer/drawerActions.ts

```diff
@@ -114,8 +120,41 @@ export const closeDrawer: ActionCreator<ThunkAction<
 
     dispatch(
       changeDrawerViewForGenome({
-        [activeGenomeId]: ''
+        [activeGenomeId]: null
       })
     );
   });
 };
+
+export const setTrackDetails = createAction('browser/set-track-details')<{
+  [genomeId: string]: { [trackId: string]: TrackDetails };
+}>();
+
+export const fetchTrackDetails: ActionCreator<ThunkAction<
```

Comment:
🙅‍♂️ `ActionCreator`. `ActionCreator` [bad](https://github.com/reduxjs/redux/blob/9d3273846aa8906d38890c410b62fb09a4992018/src/types/actions.ts#L52-L54)

Reply:
That's done.